### PR TITLE
Add onboarding documentation

### PR DESCRIPTION
**Pull request by kairstenfay**
Document the process for adding new users to the Seattle Flu Study
produciton database.

### CODE REVIEW

File: onboarding.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Seattle Flu Study onboarding
+
+## Adding new database users
+Before adding a new study member to the production database, confirm the DTUA is executed with Robin.
+
+Our convention for creating new usernames is first initial plus last name.
```
Comment (tsibley) [inline onboarding.md]: I've actually been using the username folks have at their institution so it's the same as something they're already used to. This results in a mix of slightly different username formats.

File: onboarding.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Once you've created new database credentials with the appropriate grants, send them in an encrypted email to the new study member [with the word "secure" (unquoted) in the subject](https://centernet.fredhutch.org/content/dam/centernet/u/center-it/Ignite2/Fred%20Hutch%20Email%20Encryption%20Instructions%202018-11-20.pdf).
```
Comment (tsibley) [inline onboarding.md]: Oh interesting. I was using Outlook on Office365 and clicking the Encrypt button.

File: onboarding.md (hunk 1 above)
Comment (kairstenfay) [reply onboarding.md]: Oops, I didn't do that for the most recent user.

File: onboarding.md (hunk 2 above)
Comment (kairstenfay) [reply onboarding.md]: > Might be good to also mention the `id3c user create` command.

That would be good to mention! I didn't even know it existed....

**Comment by kairstenfay** [comment]
Might be good to also mention the id3c user create command.

That would be good to mention! I didn't even know it existed....

**Comment by tsibley** [comment]
Er, how'd you create the latest user then?

**Comment by kairstenfay** [comment]
> Er, how'd you create the latest user then?

With my `kfay-admin` account, 
`create user xxx with password '...'; grant this-role to user;`

I also created Mike and Susan's accounts in this way.

**Comment by kairstenfay** [comment]
I pushed a fixup, removing the bit about our naming "convention" for users. 
I also mentioned the `id3c user` command.

**Comment by kairstenfay** [comment]
[Add onboarding documentation](https://trello.com/c/jjXePdJc/492-add-onboarding-documentation)

**Pull request merged**